libs/auth/scripts/mock_card.ts

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable no-console */
+import { Buffer } from 'buffer';
+import yargs from 'yargs/yargs';
+import { assert, throwIllegalValue } from '@votingworks/basics';
+import * as fixtures from '@votingworks/fixtures';
+import { ElectionDefinition, Optional } from '@votingworks/types';
+
+import { mockCard } from '../src/mock_file_card';
+
+const CARD_TYPES = [
+  'system-administrator',
+  'election-manager',
+  'poll-worker',
+  'unprogrammed',
+  'no-card',
+] as const;
+type CardType = typeof CARD_TYPES[number];
+
+const ELECTION_DEFINITIONS: { [electionName: string]: ElectionDefinition } = {
+  electionFamousNames2021:
+    fixtures.electionFamousNames2021Fixtures.electionDefinition,
+  electionGridLayoutNewHampshireAmherst:
+    fixtures.electionGridLayoutNewHampshireAmherstFixtures.electionDefinition,
+  electionGridLayoutNewHampshireHudson:
+    fixtures.electionGridLayoutNewHampshireHudsonFixtures.electionDefinition,
+  electionMinimalExhaustiveSample:
+    fixtures.electionMinimalExhaustiveSampleFixtures.electionDefinition,
+  electionMinimalExhaustiveSampleRightSideTargets:
+    fixtures.electionMinimalExhaustiveSampleRightSideTargetsDefinition,
+  electionMinimalExhaustiveSampleSinglePrecinct:
+    fixtures.electionMinimalExhaustiveSampleSinglePrecinctDefinition,
+  electionMinimalExhaustiveSampleWithReportingUrl:
+    fixtures.electionMinimalExhaustiveSampleWithReportingUrlFixtures
+      .electionDefinition,
+  electionMultiPartyPrimary:
+    fixtures.electionMultiPartyPrimaryFixtures.electionDefinition,
+  electionPrimary: fixtures.primaryElectionSampleFixtures.electionDefinition,
+  electionPrimaryNonpartisanContests:
+    fixtures.electionPrimaryNonpartisanContestsFixtures.electionDefinition,
+  electionSample: fixtures.electionSampleDefinition,
+  electionSample2: fixtures.electionSample2Fixtures.electionDefinition,
+  electionSampleCdf: fixtures.electionSampleCdfDefinition,
+  electionSampleLongContent: fixtures.electionSampleLongContentDefinition,
+  electionSampleNoSeal: fixtures.electionSampleNoSealDefinition,
+  electionWithMsEitherNeither:
+    fixtures.electionWithMsEitherNeitherFixtures.electionDefinition,
+};
+
+async function mockCardGivenEnvVars() {
+  const argParser = yargs()
+    .options({
+      'card-type': {
+        description: 'The type of card to mock',
+        type: 'string',
+        choices: CARD_TYPES,
+      },
+      'election-definition': {
+        description:
+          'The election definition to use for an election manager or poll worker card',
+        type: 'string',
+        choices: Object.keys(ELECTION_DEFINITIONS).sort(),
+      },
```

Comment:
I wonder if it would be easier to just allow passing in a path to an election definition like the previous script. Advantages I can think of:
- flexibly supports creating new/modified election definitions (which can happen when testing new features)
- leverages bash autocomplete for typing in a fixture name

I imagine you may have had a reason for doing it this way though, so I'm curious what that might be

Reply:
Yeah I went back and forth on this. I decided to map from strings to fixtures to avoid having to manually read files and compute election hashes, since it felt easy to get pieces of that wrong.

But thinking about it some more and prototyping some code, it's not that bad. For the benefits you mention, I'll switch over!